src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -116,7 +116,8 @@ const SubmissionDetails = _.flow(
     sort.direction === 'asc' ? _.identity : _.reverse
   )(workflows)
 
-  const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status), workflows)
+  // Note: these variable names match the id values of statusType.
+  const { succeeded, failed, running, submitted } = _.groupBy(wf => collapseStatus(wf.status).id, workflows)
```

Comment:
It would be nice if these could be declared as  `{ statusType.succeeded.id, statusType.failed.id …}`, but as far as I can tell that is not possible.